docs/docs/user/migration-guide/v4-v5.md

```diff
@@ -0,0 +1,22 @@
+# Migrate from V4 to V5
+
+## [Allow custom timeout for external services](https://github.com/bakdata/kpops/pull/485)
+
+The global `timeout` setting has been removed. Instead, an individual timeout can be set for each external service. The default is 30 seconds.
+
+#### config.yaml
+
+```diff
+- timeout: 300
```

Comment:
Removing this general timeout is the breaking change of this PR, right? What do you think about removing it in a separate PR? Then we would be able to ship the specific timeouts already in a minor KPOps release and wouldn't have to wait for this feature until v5.

Reply:
Yes, exactly. What do you mean by "wait for this feature until v5"? We will release the new major and you can update to it right away.

Afaict the global timeout was only ever used for Kafka Connect tbf 🤷🏻‍♂️

Reply:
Ah ok! I thought maybe you were planning to defer the major release to a point in the future.